### PR TITLE
Create github-ntfy.yml

### PR DESCRIPTION
**Pull request by BreizhHardware**
Thanks for taking the time to suggest an addition to awesome-selfhosted! 

To ensure your Pull Request is dealt with swiftly, please check the following (check the boxes `[x]`):
- [x] Submit one item per pull request. This eases reviewing and speeds up inclusion.
- [x] You have searched the repository for any relevant [issues](https://github.com/awesome-selfhosted/awesome-selfhosted-data/issues) or [PRs](https://github.com/awesome-selfhosted/awesome-selfhosted-data/pulls), including closed ones.
- [x] Any software you are adding is not already listed at any of [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin), [staticgen.com](https://www.staticgen.com/), [staticsitegenerators.net](https://staticsitegenerators.net/), [dbdb.io](https://dbdb.io/browse).
- [x] The file you are adding is formatted as described in [addition.md](https://github.com/awesome-selfhosted/awesome-selfhosted-data/blob/master/.github/ISSUE_TEMPLATE/addition.md).
- [x] `Demo` links should only be used for interactive demos, i.e. not video demonstrations. 
- [x] Comments and unused optional fields have been removed.
- [x] The file you are adding uses [kebab-case](https://en.wikipedia.org/wiki/Letter_case#Kebab_case) file naming, for example `my-awesome-software.yml`.
- [x] Values for `platform` are the main **server-side** requirements for the software. Don't include frameworks or specific dialects.
- [x] Any software project you are adding to the list is actively maintained.
- [x] Any software project you are adding was first released more than 4 months ago.
- [x] Any software project you are adding has working installation instructions.
- [x] You understand that your Pull Request will be merged ~1 week after approval, to allow for further comments if needed.

### CODE REVIEW

File: software/github-ntfy.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+website_url: https://github.com/BreizhHardware/ntfy_alerts
+description: Push notifications to NTFY when a new release is available on Docker Hub or Github.
+licenses:
+  - GPL-3.0
+platforms:
+  - Docker
+  - Python
+tags:
+  - Communication - Custom Communication Systems
+  - Miscellaneous
```
Comment (Ki-er) [inline software/github-ntfy.yml:11]: Not misc if it has a category
```suggestion
```

**Review by Ki-er — CHANGES_REQUESTED** [review]
Quick change

**Review by kokomo123 — APPROVED** [review]
LGTM 👍🏻 No issues here, pending other thoughts from collaborators. Thank you for your contribution!

File: software/github-ntfy.yml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+name: Github Ntfy
+website_url: https://github.com/BreizhHardware/ntfy_alerts
+description: Push notifications to NTFY when a new release is available on Docker Hub or Github.
+licenses:
+  - GPL-3.0
+platforms:
+  - Docker
+  - Python
+tags:
+  - Communication - Custom Communication Systems
```
Comment (nodiscc) [inline software/github-ntfy.yml:10]:
```suggestion
  - Automation
```

File: software/github-ntfy.yml [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+source_code_url: https://github.com/BreizhHardware/ntfy_alerts
+related_software_url: https://github.com/binwiederhier/ntfy
```
Comment (nodiscc) [inline software/github-ntfy.yml:12]:
```suggestion
related_software_url: https://github.com/binwiederhier/ntfy
depends_3rdparty: true
```

> the software depends on a third-party service outside the user's control

File: software/github-ntfy.yml [hunk 4]
```diff
@@ -0,0 +1,12 @@
+name: Github Ntfy
+website_url: https://github.com/BreizhHardware/ntfy_alerts
+description: Push notifications to NTFY when a new release is available on Docker Hub or Github.
+licenses:
+  - GPL-3.0
+platforms:
+  - Docker
+  - Python
```
Comment (nodiscc) [inline software/github-ntfy.yml:8]:
```suggestion
platforms:
  - Docker
```

Docker is the only documented installation method

**Review by nodiscc — CHANGES_REQUESTED** [review]

File: software/github-ntfy.yml (hunk 4 above)
Comment (BreizhHardware) [reply software/github-ntfy.yml:8]: Yes I remove the python one Last Week 😅

**Review by nodiscc — APPROVED** [review]

**Pull request merged**